from src.common.applications.exceptions import ApplicationError


class UserAlreadyExists(ApplicationError):
    pass
